multishot/dune

```diff
@@ -0,0 +1,29 @@
+; Disabled, see comments in memo.ml
```

Comment:
For multishot continuations, we can use https://github.com/dhil/ocaml-multicont.